doc/command_ref.rst

```diff
@@ -720,7 +720,7 @@ and :ref:`\configuration_files_replacement_policy-label`.
 Info Command
 ------------
 
-``dnf [options] info [<package-name-spec>...]``
+``dnf [options] info [<package-spec>...]``
```

Comment:
Not sure if this is good idea. It sounds like minor change but this is a huge incompatible change.